src/annotations/undo/CropCommand.cpp

```diff
@@ -25,9 +25,10 @@ CropCommand::CropCommand(QGraphicsPixmapItem *image, const QRectF &cropRect, Ann
 {
 	mAnnotationArea = annotationArea;
 	mImageItem = image;
-	mOriginalImage = QPixmap(image->pixmap());
-	mCroppedImage = QPixmap(image->pixmap().copy(cropRect.toRect()));
-	mNewItemOffset = cropRect.topLeft();
+	mOriginalImage = QPixmap(image->pixmap());    
+    QRect rect = ScaledSizeProvider::getScaledRect(cropRect.toRect(), image->pixmap().devicePixelRatio());
```

Comment:
use `auto` for local variables, like the `QRect`. Here the same, don't pass the ratio to the method, use the methods from the class.